x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_kpis/alerts_histogram_panel/index.test.tsx

```diff
@@ -530,7 +530,9 @@ describe('AlertsHistogramPanel', () => {
       wrapper.setProps({ filters: [] });
       wrapper.update();
 
-      expect(wrapper.find(`[data-test-subj="header-section-subtitle"]`).text()).toContain('999');
+      expect(wrapper.find(`[data-test-subj="header-section-subtitle"]`).first().text()).toContain(
```

Comment:
@kapral18 It seems like this file is still using enzyme. Is that because it's out of scope for the explore team?